### ISSUE TITLE
Fatal Error : Exception

### ISSUE THREAD

**Issue by rajasekharponakala** [pH7Software/pH7-Social-Dating-CMS#145]
My instance includes php7 and Mysql 5.7 and Apache. After selecting the Social Dating site option the site giving me this error.

pH7CMS Debug - Exception

Message: | 
SQLSTATE[42S02]: Base table or view not found: 1146 Table 'dtus.pH7_License' doesn't exist
-- | --
File: | 
/var/www/html/ecomm/_protected/framework/Mvc/Model/Engine/Record.class.php
Line: | 
391
Trace: | 
#0 /var/www/html/ecomm/_protected/framework/Mvc/Model/Engine/Record.class.php(391): PDOStatement->execute() 
#1 /var/www/html/ecomm/_protected/framework/Mvc/Model/License.class.php(23): PH7\Framework\Mvc\Model\Engine\Record->getOne('License', 'licenseId', 1, 'licenseKey') 
#2 /var/www/html/ecomm/_protected/framework/Core/License.class.php(147) : runtime-created function(1) : eval()'d code(148) : runtime-created function(1) : eval()'d code(108): PH7\Framework\Mvc\Model\License->get() 
#3 /var/www/html/ecomm/_protected/framework/Core/Kernel.class.php(142) : runtime-created function(1) : eval()'d code(144) : runtime-created function(1) : eval()'d code(83): PH7\Framework\Core\License->checkCopyright() 
#4 /var/www/html/ecomm/_protected/framework/Core/Kernel.class.php(142) : runtime-created function(1) : eval()'d code(144) : runtime-created function(1) : eval()'d code(68): PH7\Framework\Core\Kernel->_checkLicense() 
#5 /var/www/html/ecomm/_protected/framework/Core/Core.class.php(28): PH7\Framework\Core\Kernel->__construct() 
#6 /var/www/html/ecomm/_protected/framework/Mvc/Controller/Controller.class.php(30): PH7\Framework\Core\Core->__construct() 
#7 /var/www/html/ecomm/_protected/framework/Mvc/Router/FrontController.class.php(604): PH7\Framework\Mvc\Controller\Controller->__construct() 
#8 /var/www/html/ecomm/_protected/app/Bootstrap.php(113): PH7\Framework\Mvc\Router\FrontController->runRouter() 
#9 /var/www/html/ecomm/index.php(44): PH7\Bootstrap->run() 
#10 {main}

**Comment by pH-7** [comment]
Hi @rajasekharponakala 

Can you create a new database through your phpmyadmin and run the following files (in the same order) 

1- https://github.com/pH7Software/pH7-Social-Dating-CMS/blob/master/_install/data/sql/MySQL/pH7_SchemaGame.sql
2- https://github.com/pH7Software/pH7-Social-Dating-CMS/blob/master/_install/data/sql/MySQL/pH7_Core.sql

And let me know if you see any errors from there.

Thanks

**Comment by pH-7** [comment]
Hi @rajasekharponakala 

Can you create a new database through your phpmyadmin and run the following files (in the same order) 

* https://raw.githubusercontent.com/pH7Software/pH7-Social-Dating-CMS/master/_install/data/sql/MySQL/pH7_SchemaGame.sql
* https://raw.githubusercontent.com/pH7Software/pH7-Social-Dating-CMS/master/_install/data/sql/MySQL/pH7_Core.sql

And let me know if you see any errors from there.

Thanks

**Comment by rajasekharponakala** [comment]
Hello, 

1. First one created successfully without any error.

2. For the second one, it is showing this.

SQL Query - 

INSERT INTO pH7_Memberships (groupId, name, description, permissions, price, expirationDays, enable, orderId) VALUES
(1, 'Visitor', 'This subscription is offered to all visitors who visit the site.', 'a:24:{s:21:"quick_search_profiles";s:1:"1";s:24:"advanced_search_profiles";s:1:"1";s:10:"read_mails";s:1:"0";s:10:"send_mails";s:1:"0";s:13:"view_pictures";s:1:"1";s:15:"upload_pictures";s:1:"0";s:11:"view_videos";s:1:"1";s:13:"upload_videos";s:1:"0";s:17:"instant_messaging";s:1:"0";s:4:"chat";s:1:"1";s:12:"chatroulette";s:1:"1";s:10:"hot_or_not";s:1:"1";s:15:"love_calculator";s:1:"0";s:10:"read_notes";s:1:"1";s:11:"write_notes";s:1:"0";s:15:"read_blog_posts";s:1:"1";s:13:"view_comments";s:1:"1";s:14:"write_comments";s:1:"0";s:12:"forum_access";s:1:"1";s:19:"create_forum_topics";s:1:"0";s:19:"answer_forum_topics";s:1:"0";s:12:"games_access";s:1:"1";s:13:"webcam_access";s:1:"1";s:18:"member_site_access";s:1:"0";}', 0.00, 0, '1', 1),
(9, 'Pending', 'Pending subscription provisional migrat[...]

MySQL 

#1062 - Duplicate entry '1' for key 'PRIMARY'

**Comment by pH-7** [comment]
Thanks @rajasekharponakala 
I wasn't able to reproduce the same error on my end. It's something from your MySQL setup.

Can you create a new fresh database and execute the following SQL code and let me know if you get an error (if you get one, please copy/paste the full message here).

```sql
CREATE TABLE IF NOT EXISTS pH7_Memberships (
  groupId tinyint(2) unsigned NOT NULL AUTO_INCREMENT,
  name varchar(64) NOT NULL DEFAULT '',
  description varchar(255) NOT NULL,
  permissions text NOT NULL,
  price decimal(10,2) unsigned NOT NULL,
  expirationDays tinyint(2) unsigned NOT NULL,
  enable enum('1','0') DEFAULT '1',
  orderId tinyint(2) unsigned NOT NULL,
  PRIMARY KEY (groupId)
) ENGINE=InnoDB;

INSERT INTO pH7_Memberships (groupId, name, description, permissions, price, expirationDays, enable, orderId) VALUES
(1, 'Visitor', 'This subscription is offered to all visitors who visit the site.', 'a:24:{s:21:"quick_search_profiles";s:1:"1";s:24:"advanced_search_profiles";s:1:"1";s:10:"read_mails";s:1:"0";s:10:"send_mails";s:1:"0";s:13:"view_pictures";s:1:"1";s:15:"upload_pictures";s:1:"0";s:11:"view_videos";s:1:"1";s:13:"upload_videos";s:1:"0";s:17:"instant_messaging";s:1:"0";s:4:"chat";s:1:"1";s:12:"chatroulette";s:1:"1";s:10:"hot_or_not";s:1:"1";s:15:"love_calculator";s:1:"0";s:10:"read_notes";s:1:"1";s:11:"write_notes";s:1:"0";s:15:"read_blog_posts";s:1:"1";s:13:"view_comments";s:1:"1";s:14:"write_comments";s:1:"0";s:12:"forum_access";s:1:"1";s:19:"create_forum_topics";s:1:"0";s:19:"answer_forum_topics";s:1:"0";s:12:"games_access";s:1:"1";s:13:"webcam_access";s:1:"1";s:18:"member_site_access";s:1:"0";}', 0.00, 0, '1', 1),
(9, 'Pending', 'Pending subscription provisional migration to a different subscription.', 'a:24:{s:21:"quick_search_profiles";s:1:"1";s:24:"advanced_search_profiles";s:1:"1";s:10:"read_mails";s:1:"0";s:10:"send_mails";s:1:"0";s:13:"view_pictures";s:1:"1";s:15:"upload_pictures";s:1:"0";s:11:"view_videos";s:1:"1";s:13:"upload_videos";s:1:"0";s:17:"instant_messaging";s:1:"0";s:4:"chat";s:1:"1";s:12:"chatroulette";s:1:"1";s:10:"hot_or_not";s:1:"1";s:15:"love_calculator";s:1:"0";s:10:"read_notes";s:1:"1";s:11:"write_notes";s:1:"0";s:15:"read_blog_posts";s:1:"1";s:13:"view_comments";s:1:"1";s:14:"write_comments";s:1:"0";s:12:"forum_access";s:1:"1";s:19:"create_forum_topics";s:1:"0";s:19:"answer_forum_topics";s:1:"0";s:12:"games_access";s:1:"1";s:13:"webcam_access";s:1:"1";s:18:"member_site_access";s:1:"0";}', 0.00, 15, '0', 2),
(2, 'Regular (Free)', 'Free Membership.', 'a:24:{s:21:"quick_search_profiles";s:1:"1";s:24:"advanced_search_profiles";s:1:"1";s:10:"read_mails";s:1:"1";s:10:"send_mails";s:1:"1";s:13:"view_pictures";s:1:"1";s:15:"upload_pictures";s:1:"1";s:11:"view_videos";s:1:"1";s:13:"upload_videos";s:1:"1";s:17:"instant_messaging";s:1:"1";s:4:"chat";s:1:"1";s:12:"chatroulette";s:1:"1";s:10:"hot_or_not";s:1:"1";s:15:"love_calculator";s:1:"1";s:10:"read_notes";s:1:"1";s:11:"write_notes";s:1:"1";s:15:"read_blog_posts";s:1:"1";s:13:"view_comments";s:1:"1";s:14:"write_comments";s:1:"1";s:12:"forum_access";s:1:"1";s:19:"create_forum_topics";s:1:"1";s:19:"answer_forum_topics";s:1:"1";s:12:"games_access";s:1:"1";s:13:"webcam_access";s:1:"1";s:18:"member_site_access";s:1:"1";}', 0.00, 0, '1', 3),
(4, 'Platinum', 'The membership for the small budget.', 'a:24:{s:21:"quick_search_profiles";s:1:"1";s:24:"advanced_search_profiles";s:1:"1";s:10:"read_mails";s:1:"1";s:10:"send_mails";s:1:"1";s:13:"view_pictures";s:1:"1";s:15:"upload_pictures";s:1:"1";s:11:"view_videos";s:1:"1";s:13:"upload_videos";s:1:"1";s:17:"instant_messaging";s:1:"1";s:4:"chat";s:1:"1";s:12:"chatroulette";s:1:"1";s:10:"hot_or_not";s:1:"1";s:15:"love_calculator";s:1:"1";s:10:"read_notes";s:1:"1";s:11:"write_notes";s:1:"1";s:15:"read_blog_posts";s:1:"1";s:13:"view_comments";s:1:"1";s:14:"write_comments";s:1:"1";s:12:"forum_access";s:1:"1";s:19:"create_forum_topics";s:1:"1";s:19:"answer_forum_topics";s:1:"1";s:12:"games_access";s:1:"1";s:13:"webcam_access";s:1:"1";s:18:"member_site_access";s:1:"1";}', 9.99, 5, '1', 4),
(5, 'Silver', 'The premium membership!', 'a:24:{s:21:"quick_search_profiles";s:1:"1";s:24:"advanced_search_profiles";s:1:"1";s:10:"read_mails";s:1:"1";s:10:"send_mails";s:1:"1";s:13:"view_pictures";s:1:"1";s:15:"upload_pictures";s:1:"1";s:11:"view_videos";s:1:"1";s:13:"upload_videos";s:1:"1";s:17:"instant_messaging";s:1:"1";s:4:"chat";s:1:"1";s:12:"chatroulette";s:1:"1";s:10:"hot_or_not";s:1:"1";s:15:"love_calculator";s:1:"1";s:10:"read_notes";s:1:"1";s:11:"write_notes";s:1:"1";s:15:"read_blog_posts";s:1:"1";s:13:"view_comments";s:1:"1";s:14:"write_comments";s:1:"1";s:12:"forum_access";s:1:"1";s:19:"create_forum_topics";s:1:"1";s:19:"answer_forum_topics";s:1:"1";s:12:"games_access";s:1:"1";s:13:"webcam_access";s:1:"1";s:18:"member_site_access";s:1:"1";}', 19.99, 10, '1', 5),
(6, 'Gold', 'The must membership! The Gold!!!', 'a:24:{s:21:"quick_search_profiles";s:1:"1";s:24:"advanced_search_profiles";s:1:"1";s:10:"read_mails";s:1:"1";s:10:"send_mails";s:1:"1";s:13:"view_pictures";s:1:"1";s:15:"upload_pictures";s:1:"1";s:11:"view_videos";s:1:"1";s:13:"upload_videos";s:1:"1";s:17:"instant_messaging";s:1:"1";s:4:"chat";s:1:"1";s:12:"chatroulette";s:1:"1";s:10:"hot_or_not";s:1:"1";s:15:"love_calculator";s:1:"1";s:10:"read_notes";s:1:"1";s:11:"write_notes";s:1:"1";s:15:"read_blog_posts";s:1:"1";s:13:"view_comments";s:1:"1";s:14:"write_comments";s:1:"1";s:12:"forum_access";s:1:"1";s:19:"create_forum_topics";s:1:"1";s:19:"answer_forum_topics";s:1:"1";s:12:"games_access";s:1:"1";s:13:"webcam_access";s:1:"1";s:18:"member_site_access";s:1:"1";}', 29.99, 30, '1', 6);
```

**Comment by rajasekharponakala** [comment]
Thank you. 
Its the problem with my existing databases. So removed all and created everything fresh and now its working.

I want to know how long this trial works and what are the facilities in trail license?
can I get a developer trail license! 
Thanks again!

**Comment by pH-7** [comment]
Hi @rajasekharponakala 
Thanks for getting back to me. In order to improve the software, can you be specific on the problem your database had?

**Comment by pH-7** [comment]
Finally, the software is a fully working version (not a trial) and doesn't have limit. The only restrictions are the payment features, banners/advertising module, fake users, branding notices. Finally, you cannot use the software for  commercial purpose s(you cannot make money with your website). 
@rajasekharponakala  You can subscribe to the waiting list for the Pro version https://app.getresponse.com/site/ph7software/webform.html?wid=3783901&u=SAOV&mg_param3=4

**Comment by rajasekharponakala** [comment]
Hi, thanks.

For the pre-existed databases I have similar tables listed so I removed all of them created a fresh database and added all those tables individually. In meantime, there are few errors saying that there is no need for FOREIGN KEY if there is a PRIMARY KEY, then I removed those FOREIGN KEY listed as errors in the tables. Also, I got few errors saying that duplicate entry for PRIMARY KEY then I changed to UNIQUE in the tables which gives these kinds of errors.
